docs/docs-content/cluster-templates/modify-cluster-templates.md

```diff
@@ -0,0 +1,301 @@
+---
+sidebar_label: "Modify Cluster Templates"
+title: "Modify Cluster Templates"
+description:
+  "Learn how to modify cluster templates, including cluster profile versions and policies, and trigger manual updates."
+hide_table_of_contents: false
+sidebar_position: 20
+toc_max_heading_level: 4
+tags: ["cluster templates"]
+---
+
+:::preview
+
+:::
+
+[Cluster templates](./cluster-templates.md) use [cluster profiles](../profiles/cluster-profiles/cluster-profiles.md) and
+cluster [template policies](./create-cluster-template-policies/create-cluster-template-policies.md) to declaratively
+define and manage the software stack and lifecycle of clusters deployed with Palette. Since cluster templates are
+wrappers that reference existing profiles and policies as objects, you can modify cluster profiles and policies
+independent of cluster templates, regardless of whether the template is linked to a cluster.
+
+Each resource in Palette, such as profiles, policies, and templates, has a unique identifier (UID). When a cluster is
+deployed, it references the UID of the associated cluster template, which in turn references the UIDs of linked profiles
+and policies to define the cluster's infrastructure, applications, and operational settings.
+
+If a linked object is updated, such as changing a cluster profile version from 1.0.0 to 2.0.0, the cluster template is
+updated to reference the new UID. During the cluster's next upgrade window, it uses this updated reference to apply the
+changes. This modular architecture allows users to swap out components in a cluster template at any time, enabling
+flexible, version-driven management.
+
+![Diagram showing how cluster templates reference profiles and policies as objects](/cluster-templates_modify-cluster-templates_cluster-template-links.webp)
```

Comment:
Do we really need to explain that it links via the UID? I'd just explained that they are linked.